### DISCUSSION TITLE
First Time Contributions

### DISCUSSION THREAD

**Discussion by balex654** [hotosm/tasking-manager#6132] [Q&A]
Hello. I'm new to this repo and I want to start making some contributions. [This issue](https://github.com/hotosm/tasking-manager/issues/6128) seemed like a good place to start. I have a local branch that I'd like to push but I get a 403 error. Do I have to be given permission to push changes?

**Comment by ramyaragupathy** [comment]
Hello @balex654 - thanks for your interest! Start by:
* forking this repository
* creating a separate branch for the fix in your fork
* and then submit a PR to this repo from your fork

You can raise such discussion in [HOTOSM's slack group](http://hotosm.slack.org/). That way you will get more visibility and support from the community.

**Comment by petya-kangalova** [comment]
Hello @balex654  thanks for your interest. For the HOT Tasking Manager we also run monthly meet up sessions. You can register your [interest here ](https://forms.gle/BHz5PLGs5EvuZKfM9) next session is on 22 November .
